packages/block-editor/src/components/index.js

```diff
@@ -8,7 +8,7 @@ export { default as AlignmentToolbar } from './alignment-toolbar';
 export { default as Autocomplete } from './autocomplete';
 export { default as BlockAlignmentToolbar } from './block-alignment-toolbar';
 export { default as BlockControls } from './block-controls';
-export { default as BlockEdit } from './block-edit';
+export { default as BlockEdit, useBlockEditContext } from './block-edit';
```

Comment:
Should we add this export also to the native version of the file packages/block-editor/src/components/index.native.js?

Otherwise, headings on mobile will break.

Reply:
Yes, good catch!